### ISSUE TITLE
Cant convert Package

### ISSUE THREAD

**Issue by corincorvus** [helixarch/debtap#57]
Hi,

i tried to convert the Remote Desktop Manager debian package:
```

$ sudo debtap -u 
==> Synchronizing pkgfile database... 
:: Updating 4 repos... 
 extra is up to date 
 core is up to date 
 multilib is up to date 
 community is up to date 
==> Synchronizing debtap database... 
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                Dload  Upload   Total   Spent    Left  Speed 
100 10.0M  100 10.0M    0     0   150k      0  0:01:08  0:01:08 --:--:--  148k 
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                Dload  Upload   Total   Spent    Left  Speed 
100  112k  100  112k    0     0  96205      0  0:00:01  0:00:01 --:--:-- 96205 
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                Dload  Upload   Total   Spent    Left  Speed 
100 59366  100 59366    0     0   623k      0 --:--:-- --:--:-- --:--:--  623k 
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                Dload  Upload   Total   Spent    Left  Speed 
100 39.9M  100 39.9M    0     0  10.7M      0  0:00:03  0:00:03 --:--:-- 10.7M 
==> Downloading latest virtual packages list... 
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                Dload  Upload   Total   Spent    Left  Speed 
100   149  100   149    0     0    659      0 --:--:-- --:--:-- --:--:--   656 
100 14118    0 14118    0     0  28871      0 --:--:-- --:--:-- --:--:-- 28871 
==> Downloading latest AUR packages list... 
 % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current 
                                Dload  Upload   Total   Spent    Left  Speed 
100  414k    0  414k    0     0  1385k      0 --:--:-- --:--:-- --:--:-- 1390k 
==> Generating base group packages list... 
error: group 'base' was not found 
==> All steps successfully completed! 

$ debtap RemoteDesktopManager.Free_2020.3.1.0_amd64.deb  
==> Extracting package data... 
==> Fixing possible directories structure differencies... 
==> Generating .PKGINFO file... 

:: Enter Packager name: 
remotedesktopmanagerfree 

:: Enter package license (you can enter multiple licenses comma separated): 
1,2,3,4 

*** Creation of .PKGINFO file in progress. It may take a few minutes, please wait... 

==> Checking and generating .INSTALL file (if necessary)... 

:: If you want to edit .PKGINFO and .INSTALL files (in this order), press (1) For vi (2) For nano (3) 
For default editor (4) For a custom editor or any other key to continue: 2 

==> Generating .MTREE file... 

==> Creating final package... 
Incorrect parameters  
mv: cannot stat '*.zst': No such file or directory 
==> Package successfully created! 
==> Removing leftover files...
 
```

File 1:

```
pkgname = 
pkgver = -1 
pkgdesc = Description:RemoteDesktopManagerFree. 
url = 
builddate = 1608471720 
packager = remotedesktopmanagerfree 
arch = 
size = 213697549 
license = 1 
license = 2 
license = 3 
license = 4

```

File 2:

```
post_install() { 

       if [ ! -d "/usr/share/.mono/new-certs/Trust" ]; then 
               mkdir -p /usr/share/.mono/new-certs/Trust 
               for f in /usr/share/ca-certificates/mozilla/*.crt ; do cp $f /usr/share/.mono/new-cer> 
       fi 
} 

post_upgrade() { 
       post_install 
} 
```

What am i doing wrong?

**Comment by Rickylss** [comment]
you meet the same problem as I did. reference to #61

**Comment by helixarch** [comment]
Fixed in 3.4, closing...
